decidim-budgets/app/helpers/decidim/budgets/projects_helper.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+module Decidim
+  module Budgets
+    # A helper to render order and budgets actions
+    module ProjectsHelper
+      # Render a budget as a currency
+      #
+      # budget - A integer to represent a budget
+      def budget_to_currency(budget)
+        number_to_currency budget, unit: "€", delimiter: ".", precision: 0, format: "%n %u"
```

Comment:
As talked offline: `delimiter` and `format` options should be delegated to the locales files, as they are locale-bound, so we should remove them from here. `precision` is OK in this case. `unit` should be moved to a global configuration, like `Decidim.currency_unit`.